scout/constants/igv_tracks.py

```diff
@@ -20,8 +20,12 @@
 HG38CLINVAR_URL = "https://hgdownload.soe.ucsc.edu/gbdb/hg38/bbi/clinvar/clinvarMain.bb"
 HG19CLINVAR_URL = "https://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/clinvar/clinvarMain.bb"
 
-HG38CLINVAR_CNVS_URL = "https://hgdownload.soe.ucsc.edu/gbdb/hg38/bbi/clinvar/clinvarCnv.bb"
-HG19CLINVAR_CNVS_URL = "https://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/clinvar/clinvarCnv.bb"
+HG38CLINVAR_SVS_URL = (
+    "https://ftp.ncbi.nlm.nih.gov/pub/dbVar/sandbox/dbvarhub/hg38/clinvar_pathogenic.bb"
+)
+HG19CLINVAR_SVS_URL = (
+    "https://ftp.ncbi.nlm.nih.gov/pub/dbVar/sandbox/dbvarhubtest/hg19/clinvar_pathogenic.bb"
```

Comment:
👀  `dbvarhubtest` - one wonders if these are stable? I suppose just bc hg19, and hg38 went to the stable dir?

Reply:
Yeah! I've found the same files under: https://ftp.ncbi.nlm.nih.gov/pub/dbVar/sandbox/dbvarhub/hg19 and https://ftp.ncbi.nlm.nih.gov/pub/dbVar/sandbox/dbvarhub/hg38/, I'm going to replace the URLS with these ones, that seem more stable

Reply:
The sandbox dir doesn't seem the real thing because of its name, but that's the official address one reaches from here: https://trackhubregistry.org/search/view_trackhub/Pi-vVnIBMbWBjSAQfVVA, so it must be!

Reply:
Sandbox is stable enough for me -
that's my general code and install dir as well for stuff that doesn't follow a classical Unix model with bin, lib and so on. Like Scout. 🙃

Reply:
> that's my general code and install dir as well for stuff that doesn't follow a classical Unix model with bin, lib and so on. Like Scout

Ah then I feel confident enough to merge! 🤣